app/localization/translated/be.json

```diff
@@ -541,7 +541,8 @@
   "DefectTypesTab.defectShortNameHint": "Абрэвіятура Тыпу Дэфекту павінна мець памер ад '1' да '4' сімвалаў",
   "DefectTypesTab.deleteModalContent": "Вы ўпэўненыя, што хочаце выдаліць тып дэфекту? Усе дэфекты, якія прызнаныя '<b>{name}</b>', будуць адзначаны як '<b>{parentName}</b>'.",
   "DefectTypesTab.deleteModalHeader": "Выдаліць тып дэфекту",
-  "DefectTypesTab.description": "Тыпы Дэфектаў - гэта набор дэфектаў, якія найбольш верагодна з'явяцца падчас выканання тэстаў. Вы можаце выкарыстоўваць Тыпы Дэфектаў па змаўчанні, каб пазначыць элементы тэсту, або стварыць уласныя Тыпы Дэфектаў, каб было прасцей аналізаваць выкананне тэстаў.",
+  "DefectTypesTab.description": "Тыпы Дэфектаў - гэта набор дэфектаў, якія найбольш верагодна з'явяцца падчас выканання тэстаў. Вы можаце выкарыстоўваць Тыпы Дэфектаў па змаўчанні, каб пазначыць элементы тэсту, або стварыць уласныя Тыпы Дэфектаў, каб было прасцей аналізаваць выкананне тэстаў. Больш падрабязную інфармацыю аб Тыпах Дэфектаў вы можаце прачытаць у",
+  "DefectTypesTab.documentation": "<a>Дакументацыі</a>.",
```

Comment:
We can add <a>Дакументацыи</a> то DefectTypesTab.description

createExternalLink was created exactly for this purpose, to make it possible to avoid the of creation extra messages

Reply:
Done.